tuist/tuist
# Change the install script and `update` command to use `git ls-remote`

Based on the discussion in #3958

### Short description 📝

This PR changes how tuist determines the latest version. Rather than parsing the `Constants.swift` (for the install script) or the Changelog (for `update`), the `git ls-remote` command is now used.

### How to test the changes locally 🧐

Run the install script and the `update` command and verify that the latest version is being detected.

### Checklist ✅
